assets/css/custom.css

```diff
@@ -17,8 +17,7 @@
 }
 
 #hero .hero-bottom {
-  position: absolute;
-  bottom: 75px;
+  position: relative;
```

Comment:
Relative made a lot more sense here than absolute because of how the nav bar works, I think